x-pack/filebeat/module/suricata/eve/test/eve-small.log-expected.json

```diff
@@ -100,29 +81,18 @@
         "fileset.name": "eve",
         "http.request.method": "get",
         "http.response.body.bytes": 1155,
-        "http.response.status_code": "200",
+        "http.response.status_code": 200,
         "input.type": "log",
         "log.offset": 985,
         "network.transport": "tcp",
         "service.type": "suricata",
         "source.ip": "192.168.86.85",
         "source.port": 56119,
-        "suricata.eve.dest_ip": "192.168.86.28",
-        "suricata.eve.dest_port": 63963,
-        "suricata.eve.event_type": "http",
         "suricata.eve.flow_id": 2115002772430095,
-        "suricata.eve.http.hostname": "192.168.86.28",
         "suricata.eve.http.http_content_type": "text/xml",
-        "suricata.eve.http.http_method": "GET",
-        "suricata.eve.http.http_user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36",
-        "suricata.eve.http.length": 1155,
         "suricata.eve.http.protocol": "HTTP/1.1",
-        "suricata.eve.http.status": 200,
         "suricata.eve.http.url": "/dd.xml",
```

Comment:
suricata.eve.http.url => url.path

Reply:
I'm not so sure about that one, as we will be losing information if s.e.http.url has query and/or fragment part (those go into url.query and url.fragment).

Reply:
perhaps then url.original?

Reply:
Yeah, in this case `url.original` should be used. This is for the value as observed.

Then optionally (out of scope here) the other fields could be populated to break it down (url.path, url.query, url.fragment), or rebuild full context (url.full).

Reply:
Oh, I overlooked url.original. Done!